css/index.css

```diff
@@ -1 +1,391 @@
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+body {
+  line-height: 1.25;
+}
+ol,
+ul {
+  list-style: none;
+}
+blockquote,
+q {
+  quotes: none;
+}
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: '';
+  content: none;
+}
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+/* Set every element's box-sizing to border-box */
+* {
+  box-sizing: border-box;
+}
+html,
+body {
+  height: 100%;
+}
+body {
+  font-family: Noto Sans;
+}
+.bordered {
+  border-bottom: 1px solid black;
+}
+.flex-box {
+  display: flex;
+}
+nav {
+  display: flex;
+  justify-content: space-evenly;
+}
+nav a {
+  text-decoration: none;
+  color: #9591AF;
+  font-family: Noto Sans;
+}
+@media (max-width: 500px) {
+  nav a {
+    font-size: 0.9rem;
+  }
+}
+a:hover {
+  background: pink;
+  border-radius: 5px;
+}
+.container {
+  margin: 20px;
+}
+.heading {
+  margin: 0 auto;
+  width: 100%;
+  text-align: center;
+}
+.heading h1 {
+  font-size: 2.5rem;
+  line-height: 3;
+  font-family: Noto Sans;
+  width: 100%;
+  letter-spacing: 1.25rem;
+  color: #9591AF;
+}
+@media (max-width: 800px) {
+  .heading h1 {
+    letter-spacing: 0.55rem;
+    font-size: 1.8rem;
+  }
+}
+@media (max-width: 500px) {
+  .heading h1 {
+    letter-spacing: 0.5rem;
+    font-size: 1.35rem;
+  }
+}
+.heading .border {
+  margin: 0 auto;
+  width: 75%;
+  border: 0.5px solid #b9b9df73;
+  border-radius: 10px;
+}
+@media (max-width: 800px) {
+  .heading .border {
+    width: 70%;
+  }
+}
+.heading h2 {
+  font-family: Calibri;
+  font-size: 1.65rem;
+  line-height: 4;
+  color: #91AFA5;
+  letter-spacing: 0.25rem;
+}
+@media (max-width: 800px) {
+  .heading h2 {
+    letter-spacing: 0.22rem;
+    line-height: 4;
+    font-size: 1.25rem;
+  }
+}
+@media (max-width: 500px) {
+  .heading h2 {
+    letter-spacing: 0.22rem;
+    line-height: 5;
+    font-size: 1.1rem;
+  }
+}
+.heading .table {
+  display: table;
+  margin: 0 auto;
+}
+.heading #horizontal-list {
+  display: inline;
+  font-family: Calibri;
+  font-size: 0.75rem;
+  color: #91AFA5;
+  letter-spacing: 0.1rem;
+  list-style: none;
+}
+@media (max-width: 800px) {
+  .heading #horizontal-list {
+    letter-spacing: 0.1rem;
+    font-size: 0.7rem;
+  }
+}
+@media (max-width: 500px) {
+  .heading #horizontal-list {
+    letter-spacing: 0.1rem;
+    font-size: 0.4rem;
+  }
+}
+.heading #horizontal-list li {
+  display: inline;
+}
+.heading li::before {
```

Comment:
Nice use of ::before, this is a slightly more advanced CSS technique that Josh didn't teach us!